src/stories/Spacing.stories.tsx

```
@@ -0,0 +1,119 @@
+            This card uses 16px padding on all sides.
+          </p>
+        </div>
+      </div>
+
+      <div>
+        <h3 className="text-lg font-semibold mb-4">Buttons with 8px gap</h3>
+        <div className="flex gap-2">
+          <button className="px-4 py-2 bg-blue-500 text-white rounded">
+            Button 1
+          </button>
+          <button className="px-4 py-2 bg-blue-500 text-white rounded">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)